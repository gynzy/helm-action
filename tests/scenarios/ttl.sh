#!/bin/bash

INPUT_CLUSTERPROJECT="GKEproject" \
INPUT_CLUSTERLOCATION="GKElocation" \
INPUT_CLUSTERNAME="clusterName" \
INPUT_CLUSTERSAJSON='{"json":"here"}' \
INPUT_HELM=helm3 \
INPUT_TOKEN=foo \
INPUT_SECRETS='{"secret": "val"}' \
INPUT_CHART=app \
INPUT_NAMESPACE=default \
INPUT_RELEASE='app-pr-123' \
INPUT_VERSION=1234 \
INPUT_TRACK=stable \
INPUT_VALUES='{"replicaCount": 1, "image": {"repository": "nginx", "tag": "latest"}}' \
INPUT_TTL='7 days' \
node ../index.js
