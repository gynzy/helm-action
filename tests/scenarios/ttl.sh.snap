::debug::not setting deployment status
::debug::param: cluster_project = "GKEproject"
::debug::param: cluster_location = "GKElocation"
::debug::param: cluster_name = "clusterName"
::debug::param: cluster_sajson = "{"json":"here"}"
::debug::param: track = "stable"
::debug::param: release = "app-pr-123"
::debug::param: appName = "app-pr-123"
::debug::param: namespace = "default"
::debug::param: chart = "/usr/src/charts/app"
::debug::param: chart_version = ""
::debug::param: values = "{"replicaCount": 1, "image": {"repository": "nginx", "tag": "latest"}}"
::debug::param: dryRun = ""
::debug::param: task = ""
::debug::param: version = "1234"
::debug::param: secrets = "{"secret":"val"}"
::debug::param: valueFiles = "[]"
::debug::param: removeCanary = 
::debug::param: timeout = ""
::debug::param: repository = ""
::debug::param: atomic = "true"
::debug::param: ttl = "7 days"
::debug::param: service_account = "helm-ttl-plugin"
Setting up GKE authentication
[command]/tmp/bin/gcloud auth activate-service-account --key-file=sa.json
gcloud auth activate-service-account --key-file=sa.json
[command]/tmp/bin/gcloud container clusters get-credentials clusterName --zone GKElocation --project GKEproject
gcloud container clusters get-credentials clusterName --zone GKElocation --project GKEproject
::debug::env: KUBECONFIG="undefined"
::debug::rendering value files [./values.yml] with: {"secrets":{"secret":"val"}}
[command]/tmp/bin/helm3 upgrade app-pr-123 /usr/src/charts/app --install --namespace=default --set=app.name=app-pr-123 --set=app.version=1234 --values=./values.yml --atomic
helm3 upgrade app-pr-123 /usr/src/charts/app --install --namespace=default --set=app.name=app-pr-123 --set=app.version=1234 --values=./values.yml --atomic
Setting ttl: 7 days
[command]/tmp/bin/helm3 --namespace=default release ttl app-pr-123 --service-account=helm-ttl-plugin --set=7 days
helm3 --namespace=default release ttl app-pr-123 --service-account=helm-ttl-plugin --set=7 days
::debug::not setting deployment status
